### DISCUSSION TITLE
How can I display and simulate 3D models in .x (DirectX) file format with the Vortice library?

### DISCUSSION THREAD

**Discussion by bulent017** [amerkoleci/Vortice.Windows#431] [Q&A]
Vortice doesn't seem to support the .x file format directly. In this case, how can I convert .x files to a format compatible with Vortice or is there an alternative method to handle such files?

**Comment by amerkoleci** [comment]
Hi,
This has nothing to do with this library, I would suggest to use AssImp importer from Silk (https://www.nuget.org/packages/Silk.NET.Assimp)

Cheers
